dotnet/Razorvine.Pyrolite/Pyrolite/Pickle/Unpickler.cs

```diff
@@ -28,27 +20,12 @@ namespace Razorvine.Pickle
 [SuppressMessage("ReSharper", "InvertIf")]
 public class Unpickler : IDisposable {
 
-	protected const int HIGHEST_PROTOCOL = 4;
+	protected internal const int HIGHEST_PROTOCOL = 4;
 
 	protected readonly IDictionary<int, object> memo;
 	protected UnpickleStack stack;
 	protected Stream input;
```

Comment:
I don't believe this field is set anymore after this change.

Reply:
It's used as a version check in `Unpickler<T>`

https://github.com/adamsitnik/Pyrolite/blob/514ac386ef21196874766d31d2a5b9f4da710084/dotnet/Razorvine.Pyrolite/Pyrolite/Pickle/UnpicklerOfT.cs#L278

I did not move it to the new type because it was protected field and potentially somebody could have been using it.

Reply:
I was specifically referring to `protected Stream input;`: the line that this comment is on - line 35.

`input` no longer gets set and will always be `null`.

Reply:
@eerhardt you are right, appologies!

I saw your comment on GH and the only line changed was the one with const and for some reason I thought that you are referring to it

![obraz](https://user-images.githubusercontent.com/6011991/57066462-8743b380-6ccc-11e9-99e0-5759cd8b04d9.png)

Reply:
@irmen should we keep exposing (and setting the value of) `protected Stream input;` in the `Unpickler.cs` or it can be removed?

My understanding is that the derived type should only override the `virtual persistentLoad` method?

Reply:
Yes I think it can be removed. Possibly ``stack`` and ``memo`` can be made private as well, but that can be done in a different commit

Reply:
@irmen I have removed the field and made stack private and memo internal